extras/splat/splat-sorter.js

```diff
@@ -163,7 +188,10 @@ class SplatSorter {
                 data: oldData
             }, [oldData]);
 
-            this.vertexBuffer.setData(newData);
+            setTimeout(() => {
```

Comment:
Just wondering why the `setTimeout` is required...

Reply:
Without the setTimeout, the postMessage above will only be enqueued after setData has finished executing (setData can be slooow). With the setTimeout, postMessage is sent immediately and the worker can start sorting for the next round while setData completes.

I've confirmed this using Chrome profiler, but it's possible we can do better.

Reply:
Worthy of a comment in the code?